### ISSUE TITLE
False positive when using inline Webpack loader 

### ISSUE THREAD

**Issue by jpetitcolas** [depcheck/depcheck#236]
This tool returns a false positive when calling a script using an inline Webpack loader. For instance:

```js
require('script-loader!slick-carousel');
```

Here, `depcheck` considers `slick-carousel` as unused.

**Comment by rumpl** [comment]
not stale

**Comment by znarf** [comment]
@rumpl I would like to see this implemented, would you agree for us at Open Collective to put a bounty on it as part of our Bounty program?

**Comment by rumpl** [comment]
@znarf what's an Open Collective bounty?

**Comment by rumpl** [comment]
@jpetitcolas A year and a week later, this is fixed, will be available in the next release, next week surely
